packages/fxa-auth-server/lib/senders/emails/templates/subscriptionUpgrade/en.ftl

```diff
@@ -13,6 +13,7 @@ subscriptionUpgrade-upgrade-info = You have successfully upgraded from { $produc
 # $productPaymentCycleOld (String) - The interval of time from the end of one payment statement date to the next payment statement date of the old subscription, e.g. month
 # $paymentProrated (String) - The one time fee to reflect the higher charge for the remainder of the payment cycle, including currency, e.g. $10.00
 subscriptionUpgrade-content-charge-info = Starting with your next bill, your charge will change from { $paymentAmountOld } per { $productPaymentCycleOld } to { $paymentAmountNew } per { $productPaymentCycleNew }. At that time you will also be charged a one-time fee of { $paymentProrated } to reflect the higher charge for the remainder of this { $productPaymentCycleOld }.
+subscriptionUpgrade-content-charge-info-different-cycle = You will be charged a one-time fee of { paymentProrated } to reflect your subscription’s higher price for the remainder of this month. Starting with your next bill, your charge will change from { paymentAmountOld } per { productPaymentCycleOld } to { paymentAmountNew } per { productPaymentCycleNew %>.
```

Comment:
```suggestion
subscriptionUpgrade-content-charge-info-different-cycle = You will be charged a one-time fee of { paymentProrated } to reflect your subscription’s higher price for the remainder of this month. Starting with your next bill, your charge will change from { paymentAmountOld } per { productPaymentCycleOld } to { paymentAmountNew } per { productPaymentCycleNew }.
```

Reply:
The `$`'s are also missing in the variables (e.g.{ paymentProrated }`  -> `{ $paymentProrated }`).

Reply:
D'oh, completely missed that 🤦🏼

Reply:
Ha. Y'all are amazing. Thank you! Fixed! 👍